### DISCUSSION TITLE
Buttons "Bounce" Animation on Page (Re)Load

### DISCUSSION THREAD

**Discussion by tobiasriemenschneider** [saadeghi/daisyui#2318] [Q&A] [answered]
Hi all!
If I navigate from one page to another or also just refresh the current page, **all [Buttons](https://daisyui.com/components/button/) on the page are doing the Bounce Animation**, scaling down and up again their sizes, as if having been clicked.
**This looks a bit strange**. Any ideas? Am I doing something wrong?

**Comment by saadeghi** [comment, marked as answer]
That's intentional. It gains attention to the buttons when the page loads.
